scipy/stats/_distn_infrastructure.py

```diff
@@ -3167,29 +3176,16 @@ def expect(self, func=None, args=(), loc=0, lb=None, ub=None,
 
         Notes
         -----
-        * function is not vectorized
-        * accuracy: uses self.moment_tol as stopping criterium
-          for heavy tailed distribution e.g. zipf(4), accuracy for
-          mean, variance in example is only 1e-5,
-          increasing precision (moment_tol) makes zipf very slow
-        * suppnmin=100 internal parameter for minimum number of points to
-          evaluate could be added as keyword parameter, to evaluate functions
-          with non-monotonic shapes, points include integers in (-suppnmin,
-          suppnmin)
-        * uses maxcount=1000 limits the number of points that are evaluated
-          to break loop for infinite sums
-          (a maximum of suppnmin+1000 positive plus suppnmin+1000 negative
-          integers are evaluated)
-
-        """
-
-        # moment_tol = 1e-12 # increase compared to self.moment_tol,
-        # too slow for only small gain in precision for zipf
+        For heavy-tailed distributions, the expectation value may or may not exist,
```

Comment:
"expectation value"  either "expectation" without value or "expected value"  
the second might be better for most